utils/appConfig.ts

```diff
@@ -0,0 +1,22 @@
+class AppConfig {
+  public readonly databaseConnectionString: string = '';
+  public readonly isOnServer = typeof window === 'undefined';
+
+  constructor() {
+    this.databaseConnectionString = this.loadEnvironmentVariable(
+      'DATABASE_CONNECTION_STRING',
+    );
+  }
+
+  private loadEnvironmentVariable(key: string) {
```

Comment:
This will need tweaked once we add public env variables being exposed to the browser. This will need to take in isPublic or something similar, that way if we forget to add one it will show the proper error in the browser if it's a public env var (beginning with NEXT_PUBLIC). We could have two different app configs, but don't think that is necessary.